gradle.properties

```diff
@@ -3,11 +3,14 @@ org.gradle.jvmargs=-Xmx4g
 ## Uncomment the properties below and set their correct value.
 
 ## The Application ID displayed on the Credentials tab in the Square Application Control Panel.
-## SQUARE_READER_SDK_APPLICATION_ID=
+SQUARE_READER_SDK_APPLICATION_ID=sq0idp-tSv1iVAOJUzfzK3eUvATFg
```

Comment:
- [ ] Don't commit these testing values.  We shold also rotate the repository password 'cause you pushed it. ;-)

Reply:
Removed and invalidated!